vendor/bat-native-ledger/src/bat/ledger/internal/bitflyer/bitflyer_authorization.cc

```diff
@@ -182,7 +182,7 @@ void BitflyerAuthorization::OnClaimWallet(
                                                {}, [](type::Result) {});
 
     std::string event_text = "bitflyer";
-    if (auto wallet_ptr = GetWallet(ledger_))
+    if (auto wallet_ptr = ledger_->bitflyer()->GetWallet())
```

Comment:
nit: we could just get the wallet ptr once before `if (result == type::Result::ALREADY_EXISTS) {`

Reply:
fixed - https://github.com/brave/brave-core/pull/8566/files#diff-e9093f85b057b475c7ba0b4a068b0467e3719eb08298735736cc46e02ec558b3R179